kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/security/keystore/KeystoreService.java

```diff
@@ -135,6 +138,39 @@ public void createKeyPair(String alias, String algorithm, int keySize, String si
     public void createKeyPair(String alias, String algorithm, int keySize, String signatureAlgorithm, String attributes,
             SecureRandom secureRandom) throws KuraException;
 
+    /**
+     * Creates and persists a new keypair in the managed keystore using the specified alias
+     *
+     * @param alias
+     * @param algorithm
+     * @param algorithmParameter
+     * @param signatureAlgorithm
+     * @param attributes
+     * @param secureRandom
+     * @throws KuraException
```

Comment:
Does make sense to add some comments on what these parameters mean? The same in the methods below.

Reply:
Also, missing since annotation